src/neptune/internal/types/stringify_value.py

```diff
@@ -39,6 +39,12 @@ class StringifyValue:
     def __init__(self, value: Any):
         self.__value = value
 
+    def __len__(self) -> int:
```

Comment:
I don't like it :P. `StringifyValue` may be a single value as well and it doesn't make any sense. Think of it as some kind of a "serialization", if you need to access the underlying value -> unpack it.